test/float8/test_base.py

```diff
@@ -55,7 +55,7 @@
 torch.manual_seed(0)
 
 is_H100 = torch.cuda.is_available() and torch.cuda.get_device_capability() >= (9, 0)
-
+is_cuda_8_9 = torch.cuda.is_available() and torch.cuda.get_device_capability() >= (8, 9)
```

Comment:
should we this anywhere? It's not super clear to me which additional tests are now running relative to the original regression test job

Reply:
right, the next step would be:
1. change the tests which are gating on `IS_H100` to gate on `is_cuda_8_9` instead, in this file and all other float8 test files.  Exception: any tests which depend on rowwise gemms need to keep `IS_H100`.
2. inspect the CI logs and verify that the tests requiring this capability are no longer skippe.

Reply:
Ok, working on it

Reply:
can we delete `is_H100` if it's not used, for all of these files?

Reply:
Removed for unused files